apps/etterlatte-saksbehandling-ui/client/src/components/behandling/trygdetid/avtaler/TrygdeAvtale.tsx

```diff
@@ -163,300 +163,306 @@ export const TrygdeAvtale = ({ redigerbar }: Props) => {
           )}
         </>
       )}
-
-      {redigerbar && redigering && avtalerListe && avtaleKriterierListe && (
-        <>
-          <TrygdeAvtaleForm>
-            <FlexRow $spacing>
-              <FormWrapper>
-                <Select
-                  label="Avtale"
-                  autoComplete="off"
-                  value={trygdeavtale.avtaleKode}
-                  onChange={(e) => {
-                    if (e.target.value) {
-                      setTrygdeavtale({ ...trygdeavtale, avtaleKode: e.target.value })
-                    }
-                    velgAvtale(e.target.value)
-                  }}
-                >
-                  <TrygdetidAvtaleOptions defaultBeskrivelse="Velg avtale" trygdeavtaleOptions={avtalerListe} />
-                </Select>
-                {valgtAvtale && valgtAvtale.datoer.length > 0 && (
+      {isSuccess(hentAlleTrygdetidAvtalerRequest) &&
+        isSuccess(hentAlleTrygdetidAvtalerKriterierRequest) &&
+        isSuccess(hentTrygdeavtaleRequest) &&
+        redigerbar &&
+        redigering && (
```

Comment:
`redigerbar` og `redigering` er veldig like, kunne man kalt redigering for noe sånt som `visSkjema` eller noe tilsvarende?